exonum/src/encoding/mod.rs

```diff
@@ -12,23 +12,26 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! `encoding` is a lazy serialization library,
-//! it allows to keep struct serialized in place, and deserialize fields on demand.
+//! `encoding` is a lazy serialization library supporting zero-copy (de)serialization
+//! of primitive types, heterogeneous structures and arrays.
 //!
-//! Binary representation of structure is split into two main parts:
+//! # Structure serialization
 //!
-//! - Header - fixed sized part.
-//! - Body - dynamic sized, known only after parsing header, part.
+//! Structures are in the root of any serializable Exonum object.
+//! Binary representation of structures is split into two main parts:
 //!
-//! For easy creating this structures,
-//! in exonum you can use macros `message!` and `encoding_struct!`
-//! #Examples
-//! Imagine structure with just two field's
+//! - **Header:** a fixed-sized part
+//! - **Body:** dynamic-sized part, known only after parsing the header
```

Comment:
s/dynamic-sized/dynamically sized/? As in https://doc.rust-lang.org/beta/reference/dynamically-sized-types.html